@xen-orchestra/lite/src/views/PageUnderConstruction.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div>
```

Comment:
As stated in [docs](https://vuejs.org/api/sfc-css-features.html#scoped-style-tips), it's recommended to always use a class name, even with scoped styles.

```suggestion
  <div class="page-under-construction">
```